src/Commands/Import/ImportDatabaseCommand.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Import;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+use Terminus\Exceptions\TerminusException;
+
+class ImportDatabaseCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+    /**
+     * Imports a site archive onto a Pantheon site
+     *
+     * @authorized
+     *
+     * @command database
+     * @aliases import:database
```

Comment:
The command should be `import:database`. `database` should not direct to this command.